src/TodoList.jsx

```diff
@@ -14,34 +14,45 @@ import {
   Typography,
   Divider,
 } from "@mui/material";
-import { Delete, Edit } from "@mui/icons-material";
+import { Delete, Edit, More } from "@mui/icons-material";
 
 function ToDoList() {
-  const [task, setTask] = useState("");
+  const [isEditing, setIsEditing] = useState("");
+  const [input, setInput] = useState("");
   const [tasks, setTasks] = useState([]);
 
   const handleChange = (event) => {
-    setTask(event.target.value);
+    setInput(event.target.value);
   };
 
   const handleSubmit = () => {
```

Comment:
now this function has to consider: create new task OR edit an existing task

Reply:
Nice!